### DISCUSSION TITLE
BLACKHOLE - INSTALL DRIVE UPDATES?

### DISCUSSION THREAD

**Discussion by ccgr33n** [ExistentialAudio/BlackHole#516] [Q&A]
I don't know how to check what version I have or how to update to a later version. Also - can Blackhole be used with Big Sur or Monterey. Don't seem to find clear answer. Thank you.

**Comment by devinroth** [comment]
I’m working on a better update system.

You can check the version by looking at

/Library/Audio/Plug-Ins/HAL/BlackHole2ch.driver/Contents/Info.plist

And looking at "Bundle version string (short)”.

BlackHole will work on both Big Sur and Monterey.

**Comment by ccgr33n** [comment]
Appreciate you getting back to me.

I found: /Library/Audio/Plug-Ins/HAL/BlackHole2ch.driver/Contents/Info.plist
But nothing that gives version or shows: "Bundle version string (short)”.

I have upgraded to Monterey. Have done nothing with Blackhole, but seems to
be working. Is it necessary to update versions?

Much appreciated.

Stay safe. Remain healthy.

Best,
*~* debra
Debra Menaker
<https://www.cloudhq-mkt6.net/mail_track/link/de8ce99c0ea963e649_1642707892238?uid=241241&url=mailto%3Adeb.men%40gmail.com>

***@***.***
<https://www.cloudhq-mkt6.net/mail_track/link/de8ce99c0ea963e649_1642707892238?uid=241241&url=mailto%3ADeb.men%40gmail.com>


[image: cloudHQ]
<https://www.cloudhq-mkt6.net/mail_track/link/de8ce99c0ea963e649_1642707892238?uid=241241&url=https%3A%2F%2Fwww.free-email-tracker.com>
Powered
by
cloudHQ
<https://www.cloudhq-mkt6.net/mail_track/link/de8ce99c0ea963e649_1642707892238?uid=241241&url=https%3A%2F%2Fwww.free-email-tracker.com>

**Reply by ghost** [reply]
If you look at the first 'chunk' of text, there should be a line similar to this:

`<key>CFBundleShortVersionString</key>
	<string>x.x.x</string> `

And the x.x.x is the version number.
